invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/FileUploader/FileUploaderArea.js

```diff
@@ -95,7 +95,14 @@ const FileTableRow = ({
         />
       </Table.Cell>
       <Table.Cell data-label={i18next.t("Filename")} width={10}>
-        <div>
+        <div
+          style={{
```

Comment:
inline styling in the components is not a correct solution due to content security policy. There is already <Truncate component that you could use for this reason.

Reply:
ah I placed it here as there were quite a few places where inline styling is used in components. but ur right, I'll move it from here, The Truncate component is in react-invenio-app-ils that I guess we don't utilize in invenio-rdm-records, it could be copied over to react-invenio-forms, or the truncate from loadash can be used but that is not responsive.

Reply:
there is an equivalent of truncate in rdm, sorry I don't remember the name at the moment